axolotl-web/src/pages/Register.vue

```diff
@@ -51,19 +51,37 @@ import WarningMessage from "@/components/WarningMessage";
 import { mapState } from "vuex";
 
 export default {
-  name: "register",
+  name: "RegisterPage",
   components: {
     VueTelInput,
     WarningMessage,
   },
-  props: {
-    msg: String,
+  data() {
+    return {
+      phone: "",
+      infoPage: true,
+    };
+  },
+  computed: mapState([
+    "gui",
+    "ratelimitError",
+    "registrationStatus",
+    "captchaToken",
+    "captchaTokenSent",
+  ]),
+  mounted() {
+    var userLang = navigator.language || navigator.userLanguage;
```

Comment:
If I am understanding this correctly, `userLang` is only initiated with a value, and once assigned it does not change.
If that is correct, this would be a perfect candidate to make use of the `const` variable type to ensure that the value will not change at a later point.